iOS Example Frame/iOS Example Frame/CustomStyle1.swift

```diff
@@ -11,14 +11,14 @@ import Frames
 
 // MARK: - Color Constants -
 
-/// Predefined custom 1 theme colors
 private enum Constants {
-  static let fontColorLabel = UIColor(red: 40 / 255.0, green: 46 / 255.0, blue: 54 / 255.0, alpha: 1.0)
-  static let grayBackgroundColor = UIColor(red: 240 / 255.0, green: 245 / 255.0, blue: 249 / 255.0, alpha: 1.0)
-  static let whiteBackgroundColor = UIColor(red: 255 / 255.0, green: 255 / 255.0, blue: 255 / 255.0, alpha: 1.0)
-  static let errorLabelBackgroundColor = UIColor(red: 226 / 255.0, green: 27 / 255.0, blue: 46 / 255.0, alpha: 1.0)
```

Comment:
please apply same changes to `Style2`

Reply:
I am unsure what changes you want to apply? If using a different type of color naming logic (usage vs describing), I don't think the themes need to be in sync. After all this is demo implementation and consumer can decide how they prefer to name and structure their objects.

I would be more concerned about the verbosity of it all than UI creation patterns.

Reply:
`CustomStyle1` is refactored. `CustomStyle2` is following the same pattern for `CustomStyle1` because this the sample guide for the merchant.

Reply:
There are many ways to create and declare UI, I don't think the way I did for CustomStyle1 is any better than CustomStyle2 and do not think it needs to be identical. 

Different UIs have different problems and the way to reach their needs may be different.

@patrick-hoban-cko , @deepesh-vasthimal-cko ?

Reply:
Happy to have different 'flavours' of how the code does customisation, if there are different ways to getting to the answer

Things just need to be pretty simple - let's discuss tomorrow.